src/eventbridge/send.ts

```diff
@@ -0,0 +1,44 @@
+import AWS from 'aws-sdk';
+import EventEntry from './EventEntry';
```

Comment:
Styling suggestion: I find it more readable if all imports are destructured by default.

Reply:
Imports deconstructed were possible.